src/misc/cache/lv_image_cache.h

```diff
@@ -40,19 +42,13 @@ void lv_image_cache_drop(const void * src);
  */
 void lv_image_cache_resize(uint32_t new_size, bool evict_now);
 
-/**
- * Invalidate image header cache. Use NULL to invalidate all image headers.
- * It's also automatically called when an image is invalidated.
- * @param src pointer to an image source.
- */
-void lv_image_header_cache_drop(const void * src);
+#endif /*LV_CACHE_DEF_SIZE > 0*/
```

Comment:
How come this guard doesn't capture the last prototype in this header?

Reply:
https://github.com/lvgl/lvgl/pull/5890#pullrequestreview-1946077546 Ah, I see, please disregard.